codex/stores/blockstore.nim

```diff
@@ -88,6 +88,18 @@ method ensureExpiry*(
 
   raiseAssert("Not implemented!")
 
+method ensureExpiry*(
```

Comment:
Had to extend the `ensureExpiry` with another implementation that adapts to the Merkelization PR that was merged recently